integrations/extensions/docs/elasticsearch-install-and-setup/watsonx_discovery_install_and_setup.md

```diff
@@ -0,0 +1,396 @@
+  ```shell
+  cat <<EOF |oc apply -f -
+  apiVersion: kibana.k8s.elastic.co/v1
+  kind: Kibana
+  metadata:
+    name: ${ES_CLUSTER}
+    namespace: ${ES_NAMESPACE}
+  spec:
+    version: 8.11.1
+    count: 3
+    elasticsearchRef:
+      name: ${ES_CLUSTER}
+    podTemplate:
+      spec:
+        containers:
+        - name: kibana
+          env:
+            - name: NODE_OPTIONS
+              value: "--max-old-space-size=2048"
+          resources:
+            requests:
+              memory: 1Gi
+              cpu: 0.5
+            limits:
+              memory: 2.5Gi
+              cpu: 2
+  EOF
+  ```  
+  NOTE: the container resources are configurable.
+  * Monitor Kibana health and creation progress
+    ```shell
+    oc get kibana -n ${ES_NAMESPACE}
+    ```
+  * A ClusterIP Service is automatically created for Kibana, check it by:
+    ```shell
+    oc get service ${ES_CLUSTER}-kb-http -n ${ES_NAMESPACE} 
+    ```
+
+* Add a enterprise ECK license  
+  When you install the default distribution of ECK, you receive a Basic license. If you have a valid Enterprise 
+  subscription or a trial license extension, you will receive a link to download a license as a JSON file. You can then
+  add the license to your ECK installation. 
+  ```shell
+  oc create secret generic eck-license --from-file=my-license-file.json -n elastic-system
+  oc label secret eck-license "license.k8s.elastic.co/scope"=operator -n elastic-system
+  ```
+  NOTE: Replace `my-license-file.json` with the JSON file you have downloaded. Learn more about adding a ECK license [here](https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-licensing.html#k8s-add-license).
+
+### Verify the installation 
+* Obtain the Elasticsearch credentials
+  * The default username is `elastic`
+  * The password can be obtained by the following command:
+    ```shell
+    oc get secret ${ES_CLUSTER}-es-elastic-user -o=jsonpath='{.data.elastic}' -n ${ES_NAMESPACE}| base64 --decode; echo
+    ```
+* Use oc port-forward to access Kibana locally:
+  ```shell
+  oc port-forward service/${ES_CLUSTER}-kb-http 5601 -n ${ES_NAMESPACE}
+  ```
+* Open https://localhost:5601 in browser, and login using the Elasticsearch credentials. 
+  If you can successfully log into Kibana and see the Kibana dashboard without any errors, your ECK installation is successful.
+
+## Step 2: Enable semantic search with ELSER
+This step is to enable semantic search using ELSER. Here are the tutorials from Elasticsearch doc:  
+ELSER v1: https://www.elastic.co/guide/en/elasticsearch/reference/8.10/semantic-search-elser.html  
+ELSER v2: https://www.elastic.co/guide/en/elasticsearch/reference/8.11/semantic-search-elser.html
+
+The following steps are based on ELSER v2 model:
+### Use port-forward to access Elasticsearch cluster locally
+  ```shell
+  oc port-forward service/${ES_CLUSTER}-es-http 9200 -n ${ES_NAMESPACE}
+  ```
+
+### Create environment variables for Elasticsearch credentials
```

Comment:
Small point, but this section shows two steps.  1. Download the cert and 2. create (set) environment variables.  Since you have them both in the same section here, the two need to be linked a little better.   Step one outputs a file named tls.crt in the current directory.   But then the related environment variable references a path to a cert.   

I think if the reader reads this a couple times they'll understand that the fourth environment variable should point to the file created in step 1.  But you can make it a little easier to see how they are connected.  Consider something as simple as having the export statement be

export ES_CACERT=<path-to-your-tls.crt>